pywren/wren.py

```diff
@@ -515,7 +523,17 @@ def result(self, timeout=None, check_only=False, throw_except=True):
         self._invoke_metadata['status_done_timestamp'] = time.time()
         self._invoke_metadata['status_query_count'] = self.status_query_count
             
-        # FIXME check if it actually worked all the way through 
+        if call_status['exception'] is not None:
+            # the wrenhandler had an exception
+            exception_str = call_status['exception']
+            print(call_status.keys())
+            exception_args = call_status['exception_args']
+            if exception_args[0] == "WRONGVERSION":
```

Comment:
This is a minor nit - but it might good to put these strings `WRONGVERSION` and `OUTATIME` in a separate class (something like a enum ?)